### ISSUE TITLE
Install a supported version of drush

### ISSUE THREAD

**Issue by mjordan** [Islandora-Labs/islandora_vagrant_base_box#8]
The vagrant base box comes with drush 5.10.0, which according to [the docs](http://docs.drush.org/en/master/install/) is unsupported. However, we're [using apt-get to install drush](https://github.com/Islandora-Labs/islandora_vagrant_base_box/blob/master/scripts/drupal.sh#L21), which makes life easy. The recommended way of installing a specific version of drush is to [use composer](http://docs.drush.org/en/master/install-alternative/).

Worth it?

**Comment by ruebot** [comment]
Or we could just replace the `apt-get` with this:

```
cd /tmp
wget http://files.drush.org/drush.phar
chmod +x drush.phar
mv drush.phar /usr/local/bin/drush
```

If you want to do a pull request, I'm happy to merge, cut a release, build, export and upload it to atlas.

**Comment by mjordan** [comment]
@ruebot I did not see your response from, gulp, December (I haven't been getting all notices from Github for a long time). I'll do this over the long weekend.

**Comment by ruebot** [comment]
@mjordan oh, I wasn't trying to shame you! Just making folks aware that this has been raised as an issue. Sorry! : grimacing:

**Comment by mjordan** [comment]
Totally no problem. Sorry for the delay.

**Comment by mjordan** [comment]
No idea what's up with my notifications. I knew you replied because I saw the little '(1)' in the browser tab containing this page. No email. Yet I am getting email notifications from a bunch of repos.

**Comment by ruebot** [comment]
Weird.

**Comment by mjordan** [comment]
Even weirder - I *did* get an email notification for your last comment. Anyhooo...

**Comment by lutaylor** [comment]
Could also just do something like this:

cd /opt
git clone https://github.com/drush-ops/drush.git && cd drush && git checkout b9e6c8c00da0fbf1227869cdf915b0c6cea466cc
ln -s /opt/drush/drush /usr/bin/drush

That is if we wanted to keep consistent 6.x with travis? 

Wonder if it makes sense to keep support for the 6.x stuff for drupal7 stuff?

Just think it is important we try to stick with what is in travis for dev vms.

**Comment by ruebot** [comment]
Pull requests welcome :smile:

**Comment by mjordan** [comment]
@lutaylor's mention that Travis uses Drush 6.3.0 should probably be taken into consideration while we're working through https://jira.duraspace.org/browse/ISLANDORA-1579 and @rosiel's survey on the mailing lists. If the Vagrant stays consistent with HEAD, looks like we're sticking with a unsupported version of drush. If the community stops supporting (the officially unsupported) drush 6.x, we'll need to update the Travis configs on all repos. Are those our choices?

**Comment by rosiel** [comment]
The survey's suggesting a lot of Drush versions 5 and 6 in the wild. I think that it would be irresponsible for the community to drop support for either at this point. 

As for what we use in Travis and how that relates to what we ship with the vagrant base box... I don't know. Travis having a deprecated version is bad in principle, Travis not matching the vagrant box is bad in theory, and Travis having another variable in the testing matrix is bad in practice.

**Comment by mjordan** [comment]
```
principle != theory
theory != practice
practice != principle
```

@rosiel++ that is as cool and as useful as https://en.wikipedia.org/wiki/Project_management_triangle. Thanks!

**Comment by rosiel** [comment]
Unless it's a significant difficulty to upgrade Travis, my suggestion would be to move to Drush 7 both on Travis and the Vagrant box (because 7 is the one compatible with PHP 5.3). Let's take this community into the future! (or at least, the slightly less distant past).

**Comment by lutaylor** [comment]
https://github.com/Islandora-Labs/islandora_vagrant_base_box/pull/14
Matching drush version with what is currently in travis for 1.7 release. Travis would need to be updated and ensure no tests fail before we can move base box to drush 7. This should be considered for next release but it is out of scope of islandora_vagrant_base_box issue. We should always match community deployment guide and vagrant box drush version to what is currently in Travis. Since the 1.7 is currently going out with 6.3 the base box will have to match that.
